Automattic/jetpack
# Boost: Fix CSS proxy

#### Changes proposed in this Pull Request:
- Fix issue with CSS proxy to be able to generate critical CSS with external stylesheets

#### Jetpack product discussion
p1649638518344999-slack-C016BBAFHHS
p1649449016751239-slack-C016BBAFHHS

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Make sure your site is serving stylesheets using a CDN (i.e. Presseble)
* Try to generate Critical CSS
* Make sure it generates successfully
